run-regression-test.sh

```diff
@@ -56,7 +36,11 @@ Usage: $0 <shell_options> <framework_options>
   Optional framework_options:
      -s                                run a specified suite
      -g                                run a specified group
+     -d                                run a specified directory
      -h                                **print all framework options usage**
+     -xs                               exclude the specified suite
+     -xg                               exclude the specified group
+     -xd                               exclude the specified directory
```

Comment:
by the way, what does x mean, why did not choose -es -eg -ed to exclude the specific suite/group/directory

Reply:
`es` maybe conflict with elasticsearch, so i change to `eXclude`

Reply:
ok